modules/tinymce/src/core/test/ts/browser/options/NormalizeOptionsTest.ts

```diff
@@ -96,13 +96,13 @@ describe('browser.tinymce.core.options.NormalizeOptionsTest', () => {
       assert.equal(settings.plugins, 'a b c d', 'Should be both forced and user plugins');
     });
 
-    it('Override defaults with forced_plugins using mixed types and spaces', () => {
+    it('Override defaults with forced_plugins using strings with commas', () => {
       const defaultSettings = {
-        forced_plugins: '  a   b'
+        forced_plugins: 'a, b'
```

Comment:
Why was the whitespace removed here is it covered elsewhere since it still trims each separated string?

Reply:
Honestly, I deleted the old mixed one it's picked up that I updated and copied the one above and adjusted it so that's why it disappeared. I'll adjust one of the two to have more spaces to better test this.